katalog/calico/deploy.yml

```diff
@@ -25,31 +28,34 @@ spec:
       nodeSelector:
         kubernetes.io/os: linux
       tolerations:
-      - key: CriticalAddonsOnly
-        operator: Exists
-      - key: node-role.kubernetes.io/master
-        effect: NoSchedule
+        # Mark the pod as a critical add-on for rescheduling.
+        - key: CriticalAddonsOnly
+          operator: Exists
+        - key: node-role.kubernetes.io/master
+          effect: NoSchedule
       serviceAccountName: calico-kube-controllers
       priorityClassName: system-cluster-critical
       containers:
-      - name: calico-kube-controllers
-        image: calico/kube-controllers
-        env:
-        - name: ENABLED_CONTROLLERS
-          value: node
-        - name: DATASTORE_TYPE
-          value: kubernetes
-        livenessProbe:
-          exec:
-            command:
-            - /usr/bin/check-status
-            - -l
-          periodSeconds: 10
-          initialDelaySeconds: 10
-          failureThreshold: 6
-        readinessProbe:
-          exec:
-            command:
-            - /usr/bin/check-status
-            - -r
-          periodSeconds: 10
+        - name: calico-kube-controllers
+          image: docker.io/calico/kube-controllers:v3.23.2
```

Comment:
Maybe we can drop the tag here

Reply:
I prefer to keep the manifests aligned with upstream (i.e. with the tags), it makes the maintenance easier and lets you spot changes in upstream.

Are there any downsides that I am missing?

(commenting just here for all the other tags, will update all of them with the result of this discussion)

Reply:
gentle ping :-)

Reply:
deleted the duplicated tag from the `kustomization.yaml` file.